doc/community-build/docker/cortx-all/README.md

```diff
@@ -23,12 +23,12 @@ This document provides step-by-step instructions to build required binaries and
     
 2.  Please Checkout **kubernetes** branch for generating CORTX packages. Use below command for checkout. 
```

Comment:
checkout branch should be named as _main_

Reply:
Done. Please review.